src/components/EvModuleList.vue

```diff
@@ -25,17 +25,41 @@
     </v-expansion-panel>
     <v-expansion-panel :disabled="config_list.length == 0">
       <v-expansion-panel-header>
-        {{ config_list.length == 0 ? "No configs available" : "Available configs" }}</v-expansion-panel-header
-      >
+        {{ config_list.length == 0 ? "No configs available" : "Available configs" }}
+      </v-expansion-panel-header>
       <v-expansion-panel-content>
         <v-sheet class="d-flex align-center my-1 pa-2" elevation="1" v-for="config in config_list" :key="config">
           {{ config }}
           <v-spacer />
-          <v-btn icon @click="load_config(config)">
+          <v-btn icon @click="check_if_config_can_be_loaded(config)">
             <v-icon>mdi-upload</v-icon>
           </v-btn>
         </v-sheet>
       </v-expansion-panel-content>
+      <v-dialog
+          v-model="dialog"
+          width="auto"
+      >
+        <v-card>
+          <v-card-title class="d-flex flex-row align-baseline">
+            <v-icon
+              size="large"
+              color="error"
+            >
+              mdi-alert-circle
+            </v-icon>
+            &nbsp;
+            Warning
+          </v-card-title>
+          <v-card-text>
+            <p>Do you want to discard the current config and load the new one?</p>
+          </v-card-text>
+          <v-card-actions>
+            <v-btn color="error" @click="load_config(config_to_load)">Load config</v-btn>
+            <v-btn @click="dialog = false">Don't load config</v-btn>
+          </v-card-actions>
+        </v-card>
+      </v-dialog>
```

Comment:
I think it would make sense to put this into its own component, a general purpose "accept/discard" modal with parameters to modify the text etc.